packages/tldraw/src/lib/shapes/shared/useEditableText.ts

```diff
@@ -13,106 +13,43 @@ import { INDENT, TextHelpers } from './TextHelpers'
 export function useEditableText(id: TLShapeId, type: string, text: string) {
 	const editor = useEditor()
 	const rInput = useRef<HTMLTextAreaElement>(null)
-	const rSelectionRanges = useRef<Range[] | null>()
 	const isEditing = useValue('isEditing', () => editor.getEditingShapeId() === id, [editor])
 	const isEditingAnything = useValue('isEditingAnything', () => !!editor.getEditingShapeId(), [
 		editor,
 	])
 
 	useEffect(() => {
 		function selectAllIfEditing({ shapeId }: { shapeId: TLShapeId }) {
-			// We wait a tick, because on iOS, the keyboard will not show if we focus immediately.
-			requestAnimationFrame(() => {
-				if (shapeId === id) {
-					const elm = rInput.current
-					if (elm) {
-						if (document.activeElement !== elm) {
-							elm.focus()
-						}
-						elm.select()
-					}
-				}
-			})
+			if (shapeId === id) {
+				rInput.current?.select()
+			}
```

Comment:
you sure this is still working on iOS?

Reply:
i've been testing on the iOS sim - will definitely test on staging once it lands to double check!